style.css

```diff
@@ -0,0 +1,269 @@
+* {
+	box-sizing:	border-box;
+	font-family: "Arial";
+	font-size: 15px;
+	background-color:#000000;
+	letter-spacing: 2px;
+}
+/*TAGS*/
```

Comment:
once again, i appreciate the effort to make your code easy to understand with comments, but it is preferable in general to have no comments other than in very exceptional cases